src/libsync/propagateuploadng.cpp

```diff
@@ -346,12 +346,6 @@ void PropagateUploadFileNG::slotPutFinished()
 
     slotJobDestroyed(job); // remove it from the _jobs list
 
-    qDebug() << job->reply()->request().url() << "FINISHED WITH STATUS"
```

Comment:
You kept it in other jobs, maybe keep it here too?
Else how can you debug out all those codes?

```
src/libsync/networkjobs.cpp:    qCInfo(lcEtagJob) << "Request Etag of" << reply()->request().url() << "FINISHED WITH STATUS"
src/libsync/networkjobs.cpp:    qCInfo(lcMkColJob) << "MKCOL of" << reply()->request().url() << "FINISHED WITH STATUS"
src/libsync/networkjobs.cpp:    qCInfo(lcLsColJob) << "LSCOL of" << reply()->request().url() << "FINISHED WITH STATUS"
src/libsync/networkjobs.cpp:    qCInfo(lcPropfindJob) << "PROPFIND of" << reply()->request().url() << "FINISHED WITH STATUS"
src/libsync/networkjobs.cpp:    qCInfo(lcProppatchJob) << "PROPPATCH of" << reply()->request().url() << "FINISHED WITH STATUS"
src/libsync/networkjobs.cpp:    qCInfo(lcJsonApiJob) << "JsonApiJob of" << reply()->request().url() << "FINISHED WITH STATUS"
src/libsync/propagatedownload.cpp:            qCInfo(lcGetJob) << "GET of" << reply()->request().url().toString() << "FINISHED WITH STATUS"
src/libsync/propagateremotedelete.cpp:    qCInfo(lcDeleteJob) << "DELETE of" << reply()->request().url() << "FINISHED WITH STATUS"
src/libsync/propagateremotemove.cpp:    qCInfo(lcMoveJob) << "MOVE of" << reply()->request().url() << "FINISHED WITH STATUS"
src/libsync/propagateupload.h:        qCInfo(lcPutJob) << "PUT of" << reply()->request().url().toString() << "FINISHED WITH STATUS"
```